qa-test/src/bin/uart_uhci_async.rs

```diff
@@ -0,0 +1,102 @@
+//! This example utilises the UHCI peripheral to use UART with DMA
```

Comment:
We don't need the qa-test anymore since we have HIL tests. You can add it as a top level module doc instead like so https://github.com/esp-rs/esp-hal/blob/e42f7d2e06eac20afe0a0762677ebe9acd8043fe/esp-hal/src/i2c/master/mod.rs#L48